### PR TITLE
collects all attributes and use all validate ones in script

### PR DESCRIPTION
**Pull request by weihou**

### CODE REVIEW

File: src/meta.js [hunk 1]
```diff
@@ -2,12 +2,11 @@
...
 export function getSDKMeta() : string {
-    return base64encode(JSON.stringify({
-        url:          getScriptUrl(),
-        stageHost:    getStageHost(),
-        apiStageHost: getAPIStageHost()
-    })).replace(/\=+$/, ''); // eslint-disable-line no-useless-escape
+    const attrs = getSDKAttributes();
+
+    attrs.url = getScriptUrl();
+    return base64encode(JSON.stringify(attrs)).replace(/\=+$/, ''); // eslint-disable-line no-useless-escape
```
Comment (bluepnume) [inline src/meta.js:11]: Can we filter just `data-` attributes here?

Comment (bluepnume) [reply src/meta.js:11]: Wait, ignore this comment. `getSDKAttributes` is already doing this.

File: src/meta.js [hunk 2]
```diff
@@ -2,12 +2,11 @@
...
 
 export function getSDKMeta() : string {
-    return base64encode(JSON.stringify({
-        url:          getScriptUrl(),
-        stageHost:    getStageHost(),
-        apiStageHost: getAPIStageHost()
-    })).replace(/\=+$/, ''); // eslint-disable-line no-useless-escape
+    const attrs = getSDKAttributes();
+
+    attrs.url = getScriptUrl();
```
Comment (bluepnume) [inline src/meta.js:10]: Can we make the structure `{ url, attrs: { ... } }`

**Comment by bluepnume** [comment]
Looks great. One minor comment & will merge and publish. Thanks Wei!

File: src/meta.js (hunk 1 above)
Comment (weihou) [reply src/meta.js:11]: https://github.com/paypal/paypal-sdk-client/blob/master/src/script.js#L34
`getSDKAttributes()` only collects `data-` attributes.

File: src/meta.js (hunk 2 above)
Comment (weihou) [reply src/meta.js:10]: do you want the object will be `{
    url:          '',
    attrs:        {
        'data-stage-host':      '',
        'data-api-stage-host':  '',
        'data-merchant-id': ''
    }
}`?

Comment (bluepnume) [reply src/meta.js:10]: Yep exactly 👍

Comment (weihou) [reply src/meta.js]: fixed. also, had to fix the schema everywhere in the tests

**Pull request merged**